### ISSUE TITLE
[onnx-model] check consistence on output names

### ISSUE THREAD

**Issue by giuseppeambrosio97** [FocoosAI/focoos#118]
At the moment bisenet-former masks and logits are swapped in the onnx exported model.

Check also on other models if this error is present.

```python
    def get_dynamic_axes(self) -> DynamicAxes:
        return DynamicAxes(
            input_names=["images"],
            output_names=["logits", "masks"],
            dynamic_axes={
                "images": {0: "batch", 2: "height", 3: "width"},
            },
        )
```
